locale/en_US/admin.po

```diff
@@ -464,7 +467,7 @@ msgid "admin.cli.tool.jobs.maintenance.message"
 msgstr "Can not run queue jobs because the application maintenance mode is enabled."
 
 msgid "admin.cli.tool.jobs.available.options.list.description"
-msgstr "List all queued jobs. If you want to paginate results, use the parameters --page= and --perPage="
+msgstr "List all queued jobs. If you want to paginate results, use the parameters --page= and --perPage= . Pass flah --failed to list failied jobs."
```

Comment:
*`flah` should be removed (this reminds me of a nonsense word I use while I'm coding -- `blee`)
*`failied` should be `failed`

Reply:
Fixed.